CoreTweet.Shared/Internal/Converters.cs

```diff
@@ -248,24 +248,24 @@ public override object ReadJson(JsonReader reader, Type objectType, object exist
             switch (reader.TokenType)
             {
                 case JsonToken.String:
-                    return DateTimeOffset.ParseExact(reader.Value as string, "yyyyMMddHHmm",
-                                                     DateTimeFormatInfo.InvariantInfo,
-                                                     DateTimeStyles.AllowWhiteSpaces);
+                    return DateTime.ParseExact(reader.Value as string, "yyyyMMddHHmm",
+                                               DateTimeFormatInfo.InvariantInfo,
+                                               DateTimeStyles.AllowWhiteSpaces);
                 case JsonToken.Date:
-                    if (reader.Value is DateTimeOffset)
-                        return (DateTimeOffset)reader.Value;
+                    if (reader.Value is DateTime)
+                        return (DateTime)reader.Value;
                     else
-                        return new DateTimeOffset(((DateTime)reader.Value).ToUniversalTime(), TimeSpan.Zero);
+                        return ((DateTime)reader.Value).ToUniversalTime();
```

Comment:
The previous `if` proves that `reader.Value` is not DateTime, so this line may throw `InvalidCastException`